sdk/storage/storage-blob/src/BlobServiceClient.ts

```diff
@@ -336,6 +336,16 @@ export class BlobServiceClient extends StorageClient {
    *                                                  AnonymousCredential is used.
    * @param {StoragePipelineOptions} [options] Optional. Options to configure the HTTP pipeline.
    * @memberof BlobServiceClient
+   *
+   * @example
+   * ```js
+   * const defaultAzureCredential = new DefaultAzureCredential();
+   *
+   * const blobServiceClient = new BlobServiceClient(
```

Comment:
No examples for the other two constructor overloads?

Reply:
I'll add the example for the StorageSharedTokenCredential from the README but there is no existing sample for constructing it using a Pipeline.

Reply:
Having the code snippet for the most simplest/common overload is enough